ComfortZoneApp/app/src/main/java/com/example/comfortzone/Utils/LocationUtil.java

```diff
@@ -16,6 +16,7 @@
 import androidx.annotation.NonNull;
 import androidx.core.app.ActivityCompat;
 
+import com.example.comfortzone.GetLocationCallback;
```

Comment:
@fionamei within the project structure, its better to keep the module name in the camelcase format. So in this case instead of Utils it should be utils. I also see that there is a Fragments module, you can rename that to be fragments as well.